src/components/virus-scanner/HeroSection.tsx

```diff
@@ -41,7 +39,7 @@ const HeroSection = ({ textContent }) => {
       body: formdata,
     };
 
-    fetch(`${FILE_SCANNER_URL}/filescan`, requestOptions)
+    fetch(`https://clamav.internxt.com/filescan`, requestOptions)
```

Comment:
If we make this public, we should restrict to requests only made from "internxt.com" origin, so other people are not able to use this as free API by inverse engineering it.

Reply:
You're right, it slipped through. I must change the name of the variable...

Nice catch!